soa/maclib/SOACMGR.MAC

```diff
@@ -4,9 +4,22 @@
                &PORT=,      HOST SERVER PORT NUMBER                    X
                &SERVICES=   DEFINE NAME AND LENGTH OF SERVICE PARMS    
 *********************************************************************
-* Copyright 2006 Automated Software Tools Corporation               *
-* This source code is part of z390 assembler/emulator package       *
-* The z390 package is distributed under GNU general public license  *
+* z390 - Mainframe assembler emulator and run-time engine
+* Copyright (C) 2021 z390 Assembler LLC
+*
+* This file is part of z390.
+*
+* z390 is free software; you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation; either version 2 of the License, or
+* (at your option) any later version.
+* z390 is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License 
+* along with this program; if not, see https://www.gnu.org/licenses.
 * Author - Don Higgins                                              *
 * Date   - 12/26/06                                                 *
 *********************************************************************
```

Comment:
Two comments/observations. Ignore if you so choose.

For consistency with all other macros, should probably begin with ".*", not "*". 

Also, you've been putting the copyright block between MACRO and the prototype statement. 

If you decide to make these changes, do so for the remaining soa/maclib/*MAC members.

Reply:
Have updated as suggested.